src/site/content/en/blog/components-howto-tabs/index.md

```diff
@@ -278,12 +290,12 @@ Add `tabs.length` to make sure the index is a positive number and get the modulu
     }
 ```
 
-`_lastTab()` returns the last tab.
+`_nextTab()` gets the tab that comes after the currently selected one, wrapping around when reaching the last tab.
```

Comment:
See the [correct description](https://github.com/GoogleChromeLabs/howto-components/blob/079d0fa34ff9038b26ea8883b1db5dd6b677d7ba/elements/howto-tabs/howto-tabs.js#L229-L232).